test/integration/mon/serverspec/mon_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'serverspec'
+
+set :backend, :exec
+
+load_thres = if %w(ppc64 ppc64le).include?(os[:arch])
+               '-w 14,9,4 -c 18,13,8'
```

Comment:
This works for openpower1, but 5 and 6 are substantially more burdened:

http://munin2.osuosl.org/osuosl.org/openpower1.osuosl.org/load.html
http://munin2.osuosl.org/osuosl.org/openpower5.osuosl.org/load.html
http://munin2.osuosl.org/osuosl.org/openpower6.osuosl.org/load.html

Their long term average is 10, with regular spikes up to 30.